test/Dialect/FIRRTL/ref.mlir

```diff
@@ -1,5 +1,5 @@
 // RUN: circt-opt %s -split-input-file | circt-opt -split-input-file
-// RUN: firtool %s -split-input-file
+// RUN: firtool %s -preserve-values=named -split-input-file
```

Comment:
Using `interesting_name` and `preserve-values=named` together to prevent layer-sink from deleting instances or wires that are necessary for the test.